docs/guide/sections/dom-binding.md

```diff
@@ -1,6 +1,71 @@
-Full documentation for the DOM Binding module is coming soon.
+Cart.js ships with support for [Rivets.js][], which allows for some powerful templating approaches with minimal effort. Rivets.js allows you to describe in your markup how your data should be rendered, and then handles automatically updating the DOM when that model data changes. With Cart.js, this allows us to describe a template for our cart that will be automatically updated whenever the cart changes.
```

Comment:
"Powerful templating approaches" doesn't necessarily mean too much to users, I think this could be brought together to have a single line:

> Cart.js ships with support for Rivets.js, which allows you to describe in your markup how your data should be rendered, and then automatically handles updating the DOM when that model data changes. This allows us to describe a template for our cart that will be automatically updated whenever the cart changes.